### DISCUSSION TITLE
MAIL_TEMPLATE_DIR. Eta Error: Could not find template

### DISCUSSION THREAD

**Discussion by JustDenP** [rubiin/ultimate-nest#338] [Q&A] [answered]
Hello, I copied mailer module to my project, but it's not clear what path should I use for mail templates. Seems like 
templateDir: configService.getString('mail.templateDir') not used anywhere and only 

```const templatePath = resolve('}');```
inside sendMail function works. 

I tried both 
```typescript
const projectRoot = dirname(require.main.filename);
const templatePath = resolve(this.options.templateDir, mailOptions.template);
const templatePath = resolve(`${projectRoot}/resources/templates/reset.eta`);
```

Which gives me an absolute paths for src and dist like
Users/user/mynestproject/src/resources/templates/reset.eta
Users/user/mynestproject/dist/resources/templates/reset.eta

But both of them gives me an error
Eta Error: Could not find template Users/user/mynestproject/src/resources/templates/reset.eta

Your const ```templatePath = resolve('}');``` gives me an error 
Eta Error: Could not find template: Users/user/mynestproject/}.eta

Can you help me to figure it out? What path should I put in MAIL_TEMPLATE_DIR ? Thank you!

**Comment by rubiin** [comment, marked as answer]
@JustDenP you could try the latest codebase.
